components/TextFields/src/ContainedInputView/MDCOutlinedTextField.m

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCOutlinedTextField.h"
+
+#import <Foundation/Foundation.h>
+
+#import "private/MDCBaseTextField+MDCTextControl.h"
+#import "private/MDCTextControl.h"
+#import "private/MDCTextControlStyleOutlined.h"
+
+@interface MDCOutlinedTextField ()
+@end
+
+@implementation MDCOutlinedTextField
+
+- (instancetype)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  if (self) {
+    [self commonMDCOutlinedTextFieldInit];
+  }
+  return self;
+}
+
+- (instancetype)initWithCoder:(NSCoder *)aDecoder {
+  self = [super initWithCoder:aDecoder];
+  if (self) {
+    [self commonMDCOutlinedTextFieldInit];
+  }
+  return self;
+}
+
+- (void)commonMDCOutlinedTextFieldInit {
+  self.containerStyle = [[MDCTextControlStyleOutlined alloc] init];
+  self.borderStyle = UITextBorderStyleNone;
```

Comment:
UITextBorderStyle is something that comes with UITextField. We don't want to use it with MDCBaseTextField subclasses because it will conflict with the Material styles.

Reply:
Can we remove this line?
UITextField's default is `none` already according to the UIKit docs:
https://developer.apple.com/documentation/uikit/uitextfield/1619624-borderstyle?language=objc

also see my other comment about this API being marked NS_UNAVAILABLE.

Reply:
I think I could get rid of this 👍

Reply:
Removed the line 👍